### DISCUSSION TITLE
Creating a GitHub project?

### DISCUSSION THREAD

**Discussion by kadisonm** [sboesen/remotely-sync#92] [Ideas]
Would you consider creating a GitHub project or similar to make it more clear what is a priority, etc.

I'm having trouble with my vault syncing and am considering diving into it and seeing if I can help improve it in any way. But I was just wondering which would be of more priority or what you're planning to do next.

In terms of what I want to try and improve specifically is the sync algorithm. #40, #75, #60. It seems like there are a few issues with the current one and I'd like to try to help get these fixed since they are big tasks.

**Comment by sboesen** [comment]
Definitely appreciate the help & agree this would be useful. I'm not sure the best way to add existing open issues in bulk but I've added you to the project I just created - https://github.com/users/sboesen/projects/1

Let me know if you have more experience with projects. I think it's a great idea to communicate priority/plans! Thanks for the suggestion

**Reply by kadisonm** [reply]
Thanks I'll have a look.

**Comment by kadisonm** [comment]
I think I added all the issues, but I just made all the closed ones 'Done' and open ones 'Backlog'. Will probably have to go through it and sort them out.

**Reply by kadisonm** [reply]
Alternatively we could always just use labels? Like a priority 1-3 label. 1 being the issues most likely to be worked on? 

I'm not entirely sure how best to use projects either, I've only every used them as a really simple version of a trello board.

**Reply by sboesen** [reply]
Awesome! That works for now. I think we can remove the ready column - and looks like issues will be added [automatically](https://github.com/users/sboesen/projects/1/workflows/14935094)! 

The priority fields in projects should work for us, I think that is simple enough. 

Appreciate you going through the issues and sorting the done ones! I am going to archive the done column as it's already a lot

**Comment by sboesen** [comment]
Removed some extra views as I think they are unnecessary, more for larger teams etc. Kept assigned to me and the prioritized backlog views.

**Comment by sboesen** [comment]
Added priorities for all items. Will document my general idea on priorities in the readme.
